doc/api/tls.md

```diff
@@ -1016,7 +1053,6 @@ console.log(ciphers); // ['AES128-SHA', 'AES256-SHA', ...]
 [SSL_METHODS]: https://www.openssl.org/docs/ssl/ssl.html#DEALING_WITH_PROTOCOL_METHODS
 [tls.Server]: #tls_class_tls_server
 [SSL_CTX_set_timeout]: https://www.openssl.org/docs/ssl/SSL_CTX_set_timeout.html
-[RFC 4492]: https://www.rfc-editor.org/rfc/rfc4492.txt
```

Comment:
That's a whoops, which I'll remove. (But in my defense, that reference is no longer used. I was planning on adding a reference to the TLS-PSK RFC, but backed out because I noticed we're not doing that sort of thing anywhere.)